<?xml version="1.0" encoding="UTF-8"?><testrun name="ListsSuite" project="progfun-example" tests="6" started="6" failures="0" errors="3" ignored="0">
  <testsuite name="example.ListsSuite" time="0.06">
    <testcase name="one plus one is two" classname="example.ListsSuite" time="0.0"/>
    <testcase name="one plus two is three?" classname="example.ListsSuite" time="0.0"/>
    <testcase name="details why one plus one is not three" classname="example.ListsSuite" time="0.0"/>
    <testcase name="intNotZero throws an exception if its argument is 0" classname="example.ListsSuite" time="0.044">
      <error>org.scalatest.exceptions.TestFailedException: Expected exception java.lang.IllegalArgumentException to be thrown, but no exception was thrown.&#13;
	at org.scalatest.Assertions$class.newAssertionFailedException(Assertions.scala:318)&#13;
	at example.ListsSuite.newAssertionFailedException(ListsSuite.scala:27)&#13;
	at org.scalatest.Assertions$class.intercept(Assertions.scala:532)&#13;
	at example.ListsSuite.intercept(ListsSuite.scala:27)&#13;
	at example.ListsSuite$$anonfun$4.apply$mcV$sp(ListsSuite.scala:87)&#13;
	at example.ListsSuite$$anonfun$4.apply(ListsSuite.scala:87)&#13;
	at example.ListsSuite$$anonfun$4.apply(ListsSuite.scala:87)&#13;
	at org.scalatest.FunSuite$$anon$1.apply(FunSuite.scala:1265)&#13;
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)&#13;
	at example.ListsSuite.withFixture(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$class.invokeWithFixture$1(FunSuite.scala:1262)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)&#13;
	at org.scalatest.FunSuite$class.runTest(FunSuite.scala:1271)&#13;
	at example.ListsSuite.runTest(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)&#13;
	at scala.collection.immutable.List.foreach(List.scala:318)&#13;
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)&#13;
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)&#13;
	at org.scalatest.FunSuite$class.runTests(FunSuite.scala:1304)&#13;
	at example.ListsSuite.runTests(ListsSuite.scala:27)&#13;
	at org.scalatest.Suite$class.run(Suite.scala:2303)&#13;
	at example.ListsSuite.org$scalatest$FunSuite$$super$run(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)&#13;
	at org.scalatest.FunSuite$class.run(FunSuite.scala:1310)&#13;
	at example.ListsSuite.run(ListsSuite.scala:27)&#13;
	at org.scalatest.junit.JUnitRunner.run(JUnitRunner.scala:94)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</error>
    </testcase>
    <testcase name="sum of a few numbers" classname="example.ListsSuite" time="0.011">
      <error>scala.NotImplementedError: an implementation is missing&#13;
	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:252)&#13;
	at example.Lists$.sum(Lists.scala:26)&#13;
	at example.ListsSuite$$anonfun$5.apply$mcV$sp(ListsSuite.scala:118)&#13;
	at example.ListsSuite$$anonfun$5.apply(ListsSuite.scala:118)&#13;
	at example.ListsSuite$$anonfun$5.apply(ListsSuite.scala:118)&#13;
	at org.scalatest.FunSuite$$anon$1.apply(FunSuite.scala:1265)&#13;
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)&#13;
	at example.ListsSuite.withFixture(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$class.invokeWithFixture$1(FunSuite.scala:1262)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)&#13;
	at org.scalatest.FunSuite$class.runTest(FunSuite.scala:1271)&#13;
	at example.ListsSuite.runTest(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)&#13;
	at scala.collection.immutable.List.foreach(List.scala:318)&#13;
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)&#13;
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)&#13;
	at org.scalatest.FunSuite$class.runTests(FunSuite.scala:1304)&#13;
	at example.ListsSuite.runTests(ListsSuite.scala:27)&#13;
	at org.scalatest.Suite$class.run(Suite.scala:2303)&#13;
	at example.ListsSuite.org$scalatest$FunSuite$$super$run(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)&#13;
	at org.scalatest.FunSuite$class.run(FunSuite.scala:1310)&#13;
	at example.ListsSuite.run(ListsSuite.scala:27)&#13;
	at org.scalatest.junit.JUnitRunner.run(JUnitRunner.scala:94)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</error>
    </testcase>
    <testcase name="max of a few numbers" classname="example.ListsSuite" time="0.002">
      <error>scala.NotImplementedError: an implementation is missing&#13;
	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:252)&#13;
	at example.Lists$.max(Lists.scala:41)&#13;
	at example.ListsSuite$$anonfun$6.apply$mcV$sp(ListsSuite.scala:122)&#13;
	at example.ListsSuite$$anonfun$6.apply(ListsSuite.scala:122)&#13;
	at example.ListsSuite$$anonfun$6.apply(ListsSuite.scala:122)&#13;
	at org.scalatest.FunSuite$$anon$1.apply(FunSuite.scala:1265)&#13;
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)&#13;
	at example.ListsSuite.withFixture(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$class.invokeWithFixture$1(FunSuite.scala:1262)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.FunSuite$$anonfun$runTest$1.apply(FunSuite.scala:1271)&#13;
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)&#13;
	at org.scalatest.FunSuite$class.runTest(FunSuite.scala:1271)&#13;
	at example.ListsSuite.runTest(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.FunSuite$$anonfun$runTests$1.apply(FunSuite.scala:1304)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)&#13;
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)&#13;
	at scala.collection.immutable.List.foreach(List.scala:318)&#13;
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)&#13;
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)&#13;
	at org.scalatest.FunSuite$class.runTests(FunSuite.scala:1304)&#13;
	at example.ListsSuite.runTests(ListsSuite.scala:27)&#13;
	at org.scalatest.Suite$class.run(Suite.scala:2303)&#13;
	at example.ListsSuite.org$scalatest$FunSuite$$super$run(ListsSuite.scala:27)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.FunSuite$$anonfun$run$1.apply(FunSuite.scala:1310)&#13;
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)&#13;
	at org.scalatest.FunSuite$class.run(FunSuite.scala:1310)&#13;
	at example.ListsSuite.run(ListsSuite.scala:27)&#13;
	at org.scalatest.junit.JUnitRunner.run(JUnitRunner.scala:94)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
